README.md

```diff
@@ -4,69 +4,29 @@
 
 In this repository, you'll find a collection of [Jupyter notebooks](http://jupyter-notebook-beginner-guide.readthedocs.io/en/latest/what_is_jupyter.html) from the software developers, data scientists, and developer advocates at [Planet](https://www.planet.com/). These interactive, open-source ([APLv2](LICENSE)) guides are designed to help you explore Planet data, work with our APIs and tools, and learn how to extract information from our massive archive of high-cadence satellite imagery. We hope these guides will inspire you to ask interesting questions of Planet data. Need help? Find a bug? Please [file an issue](https://github.com/planetlabs/notebooks/issues/new) and we'll get back to you.
 
-### Available notebooks
```

Comment:
Why move this index of available notebooks to be lower down on the page?

Reply:
This was done to make sure the user sees the primary notebook installation tasks before clicking on one of the specific guides. Users might get confused clicking into the specific guides at the very top before installing the Docker fundamentals. I saw this happen with Jonathan, who was testing the README and almost clicked away into the specific guides before he realized he had to do some basic installation tasks first (Docker, etc.)